### PR TITLE
Freature #1005 - Add streaming API for Bedrock Anthropics

### PR DESCRIPTION
**Pull request by michalkozminski**
## Context
Feature adds Bedrock Antrhopics Streaming capability #1005 

Previous PR https://github.com/langchain4j/langchain4j/pull/679
## Change
Added new streaming model `AbstractBedrockStreamingChatModel`

## Checklist
Before submitting this PR, please check the following points:
- [x] I have added unit and integration tests for my change
- [x] All unit and integration tests in the module I have added/changed are green
- [x] All unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules are green
- [x] I have added/updated the [documentation](https://github.com/langchain4j/langchain4j/tree/main/docs/docs)
- [ ] I have added an example in the [examples repo](https://github.com/langchain4j/langchain4j-examples) (only for "big" features)
- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml) (only when a new module is added)

## Checklist for adding new embedding store integration
- [ ] I have added a {NameOfIntegration}EmbeddingStoreIT that extends from either EmbeddingStoreIT or EmbeddingStoreWithFilteringIT

### CODE REVIEW

**Comment by michalkozminski** [comment]
@langchain4j Previous comments were addressed. Would be amazing if you could run CI I checked locally everything compiles and tests pass.

**Comment by michalkozminski** [comment]
PR doesn't address token count as Bedrock doesn't expose such information as part of response. This will require token estimations which will be addressed in the next PR.

**Comment by michalkozminski** [comment]
@langchain4j would be amazing if you could give a look on it.

**Comment by langchain4j** [comment]
@michalkozminski I will review it asap, in the order of priorities. Thanks for your patience!

File: langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicStreamingChatModel.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import dev.langchain4j.model.bedrock.internal.AbstractBedrockStreamingChatModel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@SuperBuilder
+public class BedrockAnthropicStreamingChatModel extends AbstractBedrockStreamingChatModel {
+    @Builder.Default
+    private final String model = BedrockAnthropicStreamingChatModel.Types.AnthropicClaudeV2.getValue();
```
Comment (langchain4j) [inline langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicStreamingChatModel.java:12]: minor: should it use one of v3 models (e.g. haiku) by default?

File: README.md [hunk 2]
```diff
@@ -179,7 +179,7 @@ See example [here](https://github.com/langchain4j/langchain4j-examples/blob/main
...
-| [Amazon Bedrock](https://docs.langchain4j.dev/integrations/language-models/amazon-bedrock)         |              | ✅                                                                        |                                                                                                      | ✅                                                                   | ✅                                                                      |                                                                 |                                                                  |
+| [Amazon Bedrock](https://docs.langchain4j.dev/integrations/language-models/amazon-bedrock)         |              | ✅                                                                        | ✅                                                                                                     | ✅                                                                   | ✅                                                                      |                                                                 |                                                                  |
```
Comment (langchain4j) [inline README.md:182]: Please update https://github.com/langchain4j/langchain4j/blob/main/docs/docs/integrations/language-models/index.md as well

**Review by ghost — APPROVED** [review]
@michalkozminski thanks a lot! Just a few minor comments

**Review by ghost — APPROVED** [review]
@michalkozminski thanks a lot!

**Pull request merged**